app/controllers/api/v1/restaurants_controller.rb

```diff
@@ -1,5 +1,36 @@
 class Api::V1::RestaurantsController < ApplicationController
+
+  skip_before_action :verify_authenticity_token, :only => [:create]
+
   def show
-    render json: Restaurant.find(params[:id])
+    @restaurant = Restaurant.find(params[:id])
+    render json: @restaurant
+  end
+
+  def create
+    @restaurant = Restaurant.new
+    @restaurant.name = params[:name]
+    @restaurant.street = params[:street]
+    @restaurant.city = params[:city]
+    @restaurant.state = params[:state]
+    @restaurant.zip = params[:zip]
+    @restaurant.phone_number = params[:phoneNumber]
+    @restaurant.email = params[:email]
+    @restaurant.website = params[:website]
+    @restaurant.cuisines << Cuisine.find(params[:cuisine_id])
+    restaurant_photo = RestaurantPhoto.new
+    restaurant_photo.user = current_user
+    restaurant_photo.restaurant_photo = params[:photo]
+    restaurant_photo.restaurant = @restaurant
+    @restaurant.restaurant_photos << restaurant_photo
```

Comment:
haha come on, this is a little intense. Please use strong params here :P

Reply:
you are right, will fix

Reply:
Changed it to use strong_params. Please take this into consideration, I dont think strong params can allow embeded or nested JSON through, so I had to create additional methods in the controller to create the embedded object (like the photo for the restaurant).